src/App.tsx

```diff
@@ -1,61 +1,111 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { Todo } from './types/Todo';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todosLegacy, setTodosLegacy] = useState<Todo[]>(todos);
+  const [newTitle, setNewTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const findLargestId = () => {
+    return Math.max(...(todosLegacy.map(todo => todo.id)));
+  };
+
+  const resetForm = () => {
+    setNewTitle('');
+    setSelectedUserId(0);
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
```

Comment:
You need to add checking you can't add empty title with blank space.
<img width="555" alt="Screenshot 2022-10-31 at 18 27 05" src="https://user-images.githubusercontent.com/54940595/199059651-96fdd0ee-5f75-4f96-8c41-783602520ce6.png">